sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.netstandard2.0.cs

```diff
@@ -0,0 +1,20 @@
+namespace Azure.Monitor.Ingestion
+{
+    public partial class IngestionUsingDataCollectionRulesClient
```

Comment:
I think you need to regenerate this per https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions